lib/features/onboarding/pages/register.dart

```diff
@@ -0,0 +1,182 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:quranpally/config/config.dart';
+import 'package:quranpally/features/onboarding/widget/custom_text_form_field.dart';
+import 'package:quranpally/features/onboarding/widget/login_with_card.dart';
+import 'package:quranpally/shared/widgets/app_image.dart';
+
+import '../../../shared/widgets/regiser_login_button.dart';
+
+@RoutePage()
+class RegisterPage extends StatelessWidget {
+  const RegisterPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const Scaffold(
+      body: _RegisterPageView(),
+    );
+  }
+}
+
+class _RegisterPageView extends StatefulWidget {
+  const _RegisterPageView();
+
+  @override
+  State<_RegisterPageView> createState() => __RegisterPageViewState();
+}
+
+class __RegisterPageViewState extends State<_RegisterPageView> {
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      children: [
+        Positioned(
+          bottom: -55.h,
+          left: 0.w,
+          child: const AppImageWidget(
+            reference: AppAssets.authdownline1,
+          ),
+        ),
+        Positioned(
+          bottom: -10.h,
+          left: -25.w,
+          child: const AppImageWidget(
+            reference: AppAssets.authdownline2,
+          ),
+        ),
+        Padding(
+          padding: EdgeInsets.only(top: 90.h, left: 25.w, right: 25.w),
+          child: SizedBox(
+            height: double.infinity.h,
+            width: double.infinity.w,
```

Comment:
Use 1.sw or 1.sh if you want full decide width or height, or you can just simple use double.infinity without .h or .w